README.md

```diff
@@ -147,3 +149,55 @@ field(:create_booking, :string) do
   resolve(&TestResolver.run/3)
 end
 ```
+
+## Error Formatting
+
+The package includes GraphQL specification compliant error formatting. When enabled, validation errors from constraints or transform failures are formatted consistently.
+
+For multiple related errors, they are grouped under a single error with BAD_USER_INPUT code:
+
+```json
+{
+  "errors": [
+    {
+      "message": "Invalid input",
+      "extensions": {
+        "code": "BAD_USER_INPUT",
+        "details": {
+          "fields": [
+            {
+              "message": "min_not_met",
+              "path": ["description"],
```

Comment:
unfortunately the path is currently the field name, we'd need to figure out how to have the full path including nesting and indexing into lists here. I've seen something here:

https://hexdocs.pm/absinthe/Absinthe.Resolution.html#path/1

but its seems to be available once the schema is resolved, which happens later in the pipeline. That said, happy to look into it separately again.

Reply:
1. FYI don't use `path_string` - it doesn't work with list (positinos in list are integers and :sob:)
2. we could try making it middleware rather then pipeline; there you have access to resolution

Reply:
> FYI don't use path_string - it doesn't work with list (positinos in list are integers and 😭)

the link is indeed to [path](https://hexdocs.pm/absinthe/Absinthe.Resolution.html#path/1) 👍 

> we could try making it middleware rather then pipeline; there you have access to resolution

unfortunately these errors cause early exit in the Absinthe pipeline, i.e. they never reach the Execution phase where the middlewares are called